chrome_manifest_v3/class_model.js

```
@@ -0,0 +1,435 @@
+      return false;
+    }
+    const data = this.newCard;
+
+    if (data.useBacklink) {
+      const email = this.userEmail.replace('@', '\\@');
+      const txtDirect =
+        '[' +
+        email +
+        '](' +
+        document.location.href +
+        ' "Direct link to creator\'s email, not acccessible from anyone else")';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)